### ISSUE TITLE
why not install and reexport the reset?

### ISSUE THREAD

**Issue by mayank99** [sifferhans/css#3]
from what i can tell, you are only importing `@acab/reset.css` to manage the layer order, and most of it is left unchanged. so it might be better to import it into your index directly and set the layer. this way you will be able to get future updates to the reset.

**Comment by sifferhans** [comment]
Good call! 😁 I think I mainly copied the file in case I wanted to customize the reset a bit, but that really didn't happen.

I can always add customizations in `base.css` later if necessary
